components/ui/PageHeader.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <header class="page-header">
+    <div class="page-header__container">
+      <h1 class="page-header__title">
+        <slot />
+      </h1>
+    </div>
+  </header>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+
+@Component
+export default class extends Vue {}
+</script>
+
+<style lang="scss" scoped>
+@import '~/assets/scss/blocks/copy.scss';
+@import '~carbon-components/scss/globals/scss/typography';
+@include carbon--theme($carbon--theme--white);
+
+.page-header {
+  @include responsive-grid-bg-strip('/images/grid/grid-hero-learn.svg', auto, 28rem);
+  min-height: 28rem;
+
+  @include mq($until: large) {
+    min-height: 28rem * 40 / 64;
+  }
+
+  &__container {
+    @include contained();
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 28rem;
+
+    @include mq($until: large) {
+      height: 28rem * 40 / 64;
+    }
+  }
+
+  &__title {
+    @include type-style('expressive-heading-05', true);
+    color: $white-text-01;
+  }
+
+  &__relevant-word {
```

Comment:
To preserve the separation of concerns, the header component should not style deep content. Also, we expect the "relevant word" to become a separated component where we implement the typewriter effect. I would prefer if we convert it in a component now (adding the typewriting effect in a follow-up PR) or if we convert it in its own BEM block, with its own identity.